wallettemplate/src/main/java/wallettemplate/MainController.java

```diff
@@ -48,6 +50,7 @@
 
     private BitcoinUIModel model = new BitcoinUIModel();
     private NotificationBarPane.Item syncItem;
+    private static MonetaryFormat monetaryFormat = MonetaryFormat.BTC.noCode();
```

Comment:
Perhaps make it a true constant by adding the final keyword and uppercase the name?

Reply:
Will do.

Reply:
Done!